_posts/2020-10-10-temperature-water-content-and-polishing-effects-on-koji.md

```diff
@@ -0,0 +1,134 @@
+
+Of course, the researchers know that growing koji for a fixed time at different temperatures would not be a fair comparison. At different temperatures, the koji's growth rate (μ, measured in h<sup>-1</sup>) is also different. Thus, enzyme activity was compared at two points different points:
+
+1. When the koji reaches 10mL oxygen/g dry weight of koji, and;
+2. When the koji reaches a maximum cell mass
+
+*Conclusion:* in general, higher temperatures favour amylases, and lower temperatures favour proteases and peptidases.
+
+In practice, no koji makers would set a single temperature for their koji for the entire duration of its growth. The above temperatures are mostly important during the last 12 hours of growth when the mycelium is producing the most enzymes.
+
+As an aside, this was the experimental set-up:
+
+![Figure 1](/assets/images/temp_effect/Figure 1.png){:.responsive .center}
+*The experimental setup - see Ref. 2*
+
+*Callout:* This setup is a sort of respirometer - I'm not sure of the exact name. Oxygen is produced by electrolysis, which builds up pressure in the system, pushing one side of the mercury manometer down. With enough pressure, the mercury level switch opens, stopping the generation of oxygen. The koji continues consuming oxygen, and one can measure total oxygen consumption by integrating the total current consumed over time. Finally, waste carbon dioxide is absorbed by the NaOH solution.
+
+### Water content
+
+Another parameter they tested is the **steamed rice water absorption rate**, which is defined as follows:
+
+$$
+\text{Steamed rice water absorption rate (%)} = \frac{\text{Steamed rice weight} - \text{White rice weight}}{\text{White rice weight} }   \times 100\%
+$$
+
+Note that this is NOT the same as total water content, since the dry white rice also contains a small amount of water (around 15% by weight). When varying the steamed rice water absorption rate from 24 - 51%, this is what they found:
+
+![Table 2](/assets/images/temp_effect/Table 2.png)
+*Effect of water absorption rate on enzyme activity*
+
+The researchers used the same metric above, and pulled the koji once it had consumed 10mL oxygen / g koji for a fair comparison. Here, the total culture time is an indicator of growth rate: the shorter the culture time, the faster the growth rate.
+
+*Conclusion:* although wetter rice will grow koji faster, it yields less enzymes in total (both amylase and proteases).
+
+Koji makers already knew that overly wet rice yielded weak koji. Another researcher found that wet rice lead to the over-accumulation of glucose, which inhibits enzyme production. In practice, the water content actually changes throughout the koji production period as some of the water vapour is released from the rice. Typical operations start at 35-40% steamed rice water absorption rate, and finish at 24 - 28% (mirin) down to 17-19% (sake).
```

Comment:
> Another researcher found that wet rice lead to the over-accumulation of glucose, which inhibits enzyme production.

sounds like a case of "positive transcription control", similar to what happens under **Can we affect the amount of lactase produced?** here: https://controlledmold.com/why-does-koji-make-lactase-anyways/

Reply:
Absolutely this is internal feedback inhibition